### PR TITLE
Adding license file and license info to package.json

### LINKED ISSUE #32: Please add a LICENSE file
Please add a LICENSE file so we can know what is permitted and what is not. Thank you!

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #32

### CODE REVIEW

**Comment by vladikoff** [comment]
@pdehaan why are there conflicts?

**Comment by pdehaan** [comment]
@vladikoff Apparently my package.json was slightly out of date and crashed on impact.
Rebased and force pushed.

File: package.json [hunk 1]
```diff
@@ -1,26 +1,29 @@
 {
+  "name": "lockdown",
+  "description": "Lock your node.js app to specific versions (and checksums) of dependencies.",
+  "version": "0.0.7-dev",
   "author": "Lloyd Hilaiel <lloyd@hilaiel.com> (http://lloyd.io)",
+  "bin": {
+    "lockdown": "./lockdown.js",
+    "lockdown-relock": "./relock.js"
+  },
+  "bugs": "https://github.com/mozilla/node-convict/issues",
```
Comment (jrgm) [inline package.json]: node-convict?

File: package.json [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
     "osenv": "0.1.0",
     "read-installed": "3.1.0"
   },
-  "bin": {
-    "lockdown": "./lockdown.js",
-    "lockdown-relock": "./relock.js"
+  "engines": {
+    "node": ">=0.6.17"
+  },
+  "homepage": "https://github.com/mozilla/node-convict",
```
Comment (jrgm) [inline package.json]: and node-convict again

Comment (vladikoff) [inline package.json]: fixed! Thanks!

**Pull request merged**